setup.py

```diff
@@ -96,6 +99,8 @@ def get_version():
         namespace=gitHubNamespace, projectName=projectName
     )
 )
+requirements = list(dict.fromkeys(get_requirements(requirementsFile)))
```

Comment:
If the order of requirements is not significant (I don't think it is?), `list(set(get_requirements(requirementsFile)))` might be a bit clearer here.

Reply:
for new Python versions (so starting around 3.6), `list(set(...))` is not recommended anymore.

Reply:
Interesting, do you have a source for that? I can't really imagine `dict.fromkeys` being more idiomatic than `set` even though there is no dict involved.

Reply:
Thanks @Xiretza. I did tests with `timeit.repeat(...)` myself and the articles about list deduplication are not correct.
A modified version has been force-pushed.